### PR TITLE
doc: Add Prometheus troubleshooting

### PR DESCRIPTION
**Pull request by michalborkowski96**
#### Summary
Closes https://github.com/TheThingsIndustries/lorawan-stack-support/issues/537

#### Changes
Added troubleshooting section per issue

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/getting-started/aws/ecs/monitoring/_index.md [hunk 1]
```diff
@@ -95,3 +95,11 @@ For some of these metrics, there are also variants with `:avg` and `:stddev` suf
 For the Join Server:
 
 - `ttn_lw_js_join_accepted_rate:by_tenant_id` and `ttn_lw_js_join_rejected_rate:by_tenant_id` record the rate of accepted and rejected join requests.
+
+## Troubleshooting
+
+Sometimes monitoring service fails with `Essential container in task exited` error. This is often caused by insufficient resources. Therefore you can:
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/monitoring/_index.md:101]: `Sometimes the monitoring`

File: doc/content/getting-started/aws/ecs/monitoring/_index.md [hunk 2]
```diff
@@ -95,3 +95,11 @@ For some of these metrics, there are also variants with `:avg` and `:stddev` suf
...
+
+1. Set `UseLongTermStorage` to `false` in order to disable Thanos and allow Prometheus use more of assigned resources.
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/monitoring/_index.md:103]: `the assigned resources.`

File: doc/content/getting-started/aws/ecs/monitoring/_index.md [hunk 3]
```diff
@@ -95,3 +95,11 @@ For some of these metrics, there are also variants with `:avg` and `:stddev` suf
...
+
+2. Increase memory available to the Prometheus task. Value of 2GB should be sufficient in almost all cases.
```
Comment (KrishnaIyer) [inline doc/content/getting-started/aws/ecs/monitoring/_index.md:105]: `A value of 2GB`

**Review by KrishnaIyer — APPROVED** [review]
Only minor grammatical stuff.

**Comment by michalborkowski96** [comment]
Fixed

**Pull request merged**